hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -127,24 +127,6 @@
       stage: V2
       tests: ["i2c_host_override"]
     }
-    {
-      name: host_fifo_watermark
-      desc: '''
-            Test the watermark interrupt of fmt_fifo and rx_fifo.
-
-            Stimulus:
-              - Configure DUT/Agent to Host/Target mode respectively
-              - Program random fmt_fifo and rx_fifo watermark level
-              - Write data quickly to fmt_fifo and rx_fifo for triggering watermark interrupts
-
-            Checking:
-              - Ensure the fmt_fifo and rx_fifo watermark interrupts are asserted
-              - Ensure the fmt_fifo and rx_fifo watermark interrupts stay asserted until cleared
-              - Ensure receving correct number of fmt_fifo and rx_fifo watermark interrupts
-            '''
-      stage: V2
-      tests: ["i2c_host_fifo_watermark"]
```

Comment:
Sorry for the confusion, what I meant was not to removed the `host_fifo_watermark` entry but to replace this line with:
`tests: [""]`

Reply:
Updated as per comments